### DISCUSSION TITLE
What is the limit on vehicle location requests

### DISCUSSION THREAD

**Discussion by xevg** [Hyundai-Kia-Connect/hyundai_kia_connect_api#376] [General]
I see there is something in the code with limits for get_vehicle_location rate, but do you know what it is? I'm trying to understand the comment: "Only update the location if the odometer moved AND if the last location update was over an hour ago." Does this mean that I will get that error if I try more than once an hour, and if the odometer hasn't moved?

Also, is the API and these limits documented anywhere? I want to understand all the underlying functionality, but I can't find any documentation anywhere.

**Comment by cdnninja** [comment]
Here is what we know on limits api side.  

https://github.com/Hacksore/bluelinky/wiki/API-Rate-Limits

Which region are you referencing this on?  Each one's a bit different.

**Reply by xevg** [reply]
I am in the US. Is the 30 a day for all requests, or commands, or locations?
